less/feed.less

```diff
@@ -2,7 +2,7 @@
   display: block;
   padding-top: 10px;
   height: 100%;
-  background-color: rgba(229, 229, 229, 0.3);
+  background-color: #dcdcdc;
```

Comment:
What's the purpose of this change? To avoid rgba codes? The previous value uses alpha chanel. Is this chnage necessary?